### ISSUE TITLE
[Feature Request] textVisiblePassword (Android)

### ISSUE THREAD

**Issue by yuridiniz** [flutter/flutter#31738]
On Android it is possible to get a Keyboard with letters and numbers visible at the same time through the property `textVisiblePassword`

In Flutter, we use a TextField with the property `obscureText: true` to get the password behavior, so there is no `TextInputType` relative to the password field, so we can not achieve the same effect when using the `textVisiblePassword` property

## Sample

```dart
TextField(controller: textController, keyboardType: TextInputType.textWithNumber)
```

## Expected behavior

![keyboard_example](https://user-images.githubusercontent.com/5683101/56855166-03c74280-6919-11e9-9e84-ca5088ec8e48.png)

**Comment by justinmc** [comment]
Sounds like this would be a good addition to Flutter to me.  We would need to add support for [InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD](https://developer.android.com/reference/android/text/InputType#TYPE_TEXT_VARIATION_VISIBLE_PASSWORD) into [TextInputPlugin.java](https://github.com/flutter/engine/blob/master/shell/platform/android/io/flutter/plugin/editing/TextInputPlugin.java#L81) in the engine, and then tie it through to [TextInputType](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/services/text_input.dart#L25) in the framework.  Should be a straightforward change.

**Comment by niktses** [comment]
Is there any update on this? I really need it to release my app

**Comment by justinmc** [comment]
@niktses Support was added in the engine in https://github.com/flutter/engine/pull/9999, and this PR will add the input type to the framework when it's merged: https://github.com/flutter/flutter/pull/36695

**Comment by niktses** [comment]
Thanks a lot

**Comment by justinmc** [comment]
Support has been added with the PR referenced above.  Make sure you're on the master [channel](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels) or you have the next version to be released after 1.8.2.  Then you can do this:

```dart
TextField(
  keyboardType: TextInputType.visiblePassword,
),
```
